src/test/cucumber/de/filefighter/rest/crudPermissions.feature

```diff
@@ -0,0 +1,35 @@
+Feature: CRUD Permissions
+  The owner of a file want to be able to give or revoke other users permissions to either see or see and edit certain files or folders, so they can work together on the same files.
+
+Scenario Outline: Successful interaction for changing existing permission
+  Given credential are valid
+  And user 1234 is owner of <type> <path>
+  And user 9877 was given permisssion of  <old_permission> for <type> <path>
+  When user 1234 wants to change permissions of <type> <path>
+  Then list all users that have permissions for <type> <path>
```

Comment:
`Then` should be implementable with a `assertThat` or `assertTrue` etc..

Reply:
makes sense